app/stac_api/migrations/0024_added_update_interval_field.py

```diff
@@ -0,0 +1,132 @@
+            field=models.IntegerField(
+                default=-1,
+                help_text=
+                'Interval in seconds in which the asset data is updated.-1 means that the data is not on a regular basis updated.',
+                validators=[django.core.validators.MinValueValidator(-1)]
+            ),
+        ),
+        migrations.AddField(
+            model_name='assetupload',
+            name='update_interval',
+            field=models.IntegerField(
+                default=-1,
+                help_text=
+                'Interval in seconds in which the asset data is updated.-1 means that the data is not on a regular basis updated.This field is can only be set via the API.',
+                validators=[django.core.validators.MinValueValidator(-1)]
+            ),
+        ),
+        migrations.AddField(
+            model_name='collection',
+            name='update_interval',
+            field=models.IntegerField(
+                default=-1,
+                help_text=
+                'Minimal update interval in seconds in which the underlying assets data are updated.',
+                validators=[django.core.validators.MinValueValidator(-1)]
+            ),
+        ),
+        migrations.AddField(
+            model_name='item',
+            name='update_interval',
+            field=models.IntegerField(
+                default=-1,
+                help_text=
+                'Minimal update interval in seconds in which the underlying assets data are updated.',
+                validators=[django.core.validators.MinValueValidator(-1)]
+            ),
+        ),
+        migrations.AddIndex(
+            model_name='item',
+            index=models.Index(fields=['update_interval'], name='item_update_interval_idx'),
+        ),
+        pgtrigger.migrations.AddTrigger(
+            model_name='asset',
+            trigger=pgtrigger.compiler.Trigger(
+                name='add_del_asset_item_update_interval_trigger',
+                sql=pgtrigger.compiler.UpsertTriggerSql(
+                    declare=
+                    'DECLARE asset_instance stac_api_asset%ROWTYPE; item_update_interval RECORD;',
+                    func=
+                    "\n        asset_instance = COALESCE(NEW, OLD);\n\n        -- Compute item update_interval (minimum aggregation of asset's update_interval)\n        SELECT\n            COALESCE(MIN(NULLIF(asset.update_interval, -1)), -1) AS min_update_interval\n        INTO item_update_interval\n        FROM stac_api_asset AS asset\n        WHERE asset.item_id = asset_instance.item_id;\n\n        -- Update related item update_interval variables\n        UPDATE stac_api_item SET\n            update_interval = item_update_interval.min_update_interval\n        WHERE id = asset_instance.item_id;\n\n        RAISE INFO 'item.id=% update_interval updated, due to asset.name=% updates.',\n            asset_instance.item_id, asset_instance.name;\n\n        RETURN asset_instance;\n        ",
```

Comment:
OK, I guess this answers my question from above. On item level it seems to be the minimum value of all the asses of that collection. And on collection level the minimum of all items.